### PR TITLE
Book component

### PR DESCRIPTION
**Pull request by mcolombo87**
## Summary

Kick off, first PR of the project. The book component was created.

## Screenshots
**Nexus 5X de 5.2" Android 10 (API 29)**
image

**Pixel 3 XL 6.3 PIE (9)**
image

**Pixel 2 de 5" Nogaout (7.1)**
image

**Pantalla 5.4" Lollipop (5.1)**
image

## Trello Card

https://trello.com/c/Nrpfqqe8/10-componente-libro

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "name": "WBooks",
+  "displayName": "WBooks"
+}
```
Comment (guidoprinc) [inline app.json:4]: add a new line

File: src/app/App.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styles from './styles';
+import Library from './screens/Library';
+
+declare const global: {HermesInternal: null | {}};
+
+const App = () => {
+  return (
+    <Library
```
Comment (guidoprinc) [inline src/app/App.tsx:9]: i would change the name to be more like it as what it is in the app, a book item

File: src/app/screens/Library/index.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import {Text, View, Image} from 'react-native';
+import styles from './styles';
+
+export interface Props {
+  title: string;
+  author: string;
+  uri: object;
+}
+
+class Library extends Component<Props> {
```
Comment (guidoprinc) [inline src/app/screens/Library/index.tsx:11]: we don't use components anymore and instead use arrow functions so we can use hooks. Please change this

File: src/app/screens/Library/index.tsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  render() {
+    return (
+      <View style={styles.bookContainer}>
+        <Image source={this.props.uri} style={styles.cover} />
+        <View style={styles.description}>
+          <Text style={styles.title}>{this.props.title}</Text>
+          <Text style={styles.author}>{this.props.author}</Text>
```
Comment (guidoprinc) [inline src/app/screens/Library/index.tsx:18]: destruct all props

File: src/app/styles.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+import {StyleSheet} from 'react-native';
+
+export default StyleSheet.create({});
```
[the remark this answers is not in the document]
Comment (mcolombo87) [reply src/app/styles.js]: Done

File: app.json (hunk 1 above)
Comment (mcolombo87) [reply app.json]: Done

File: src/app/App.tsx (hunk 2 above)
Comment (mcolombo87) [reply src/app/App.tsx]: Done

File: src/app/screens/Library/index.tsx (hunk 3 above)
Comment (mcolombo87) [reply src/app/screens/Library/index.tsx]: Done

File: src/app/screens/Library/index.tsx (hunk 4 above)
Comment (mcolombo87) [reply src/app/screens/Library/index.tsx]: Done

**Pull request merged**